File: January 2022/Day 28/Que6.java
// https://www.codechef.com/problems/CHFRICH
// Problem Code: CHFRICH


/* package codechef; // don't place package name! */

import java.util.*;
import java.lang.*;
import java.io.*;

/* Name of the class has to be "Main" only if the class is public. */
class Codechef
{
	public static void main (String[] args) throws java.lang.Exception
	{
		// your code goes here
		Scanner scan = new Scanner(System.in);
        int T = scan.nextInt();
        while(T-- != 0){
            int A = scan.nextInt();
            int B = scan.nextInt();
            int X = scan.nextInt();
            System.out.println((B-A)/X);
        }
	}
}

File: January 2022/Day 28/Que6.py
# https://www.codechef.com/problems/CHFRICH
# Problem Code: CHFRICH


# cook your dish here
T = int(input())
for _ in range(T):
    A, B, X = list(map(int, input().rstrip().split()))
    print(int((B-A)/X))